cylc/flow/cfgspec/suite.py

```diff
@@ -323,28 +324,37 @@ def upg(cfg, descr):
     # and [job] moved up 1 level for example) which should be upgraded here.
     u.upgrade()
 
-    # Upgrader cannot do this type of move.
-    try:
-        keys = set()
-        cfg['scheduling'].setdefault('graph', {})
-        cfg['scheduling']['graph'].update(
-            cfg['scheduling'].pop('dependencies'))
-        graphdict = cfg['scheduling']['graph']
-        for key, value in graphdict.copy().items():
-            if isinstance(value, dict) and 'graph' in value:
-                graphdict[key] = value['graph']
-                keys.add(key)
-        if keys:
-            LOG.warning(
-                "deprecated graph items were automatically upgraded in '%s':",
-                descr)
-            LOG.warning(
-                ' * (8.0.0) %s -> %s - for X in:\n%s',
-                u.show_keys(['scheduling', 'dependencies', 'X', 'graph']),
-                u.show_keys(['scheduling', 'graph', 'X']),
-                '\n'.join(sorted(keys)),
-            )
-
+    # Upgrader cannot do this type of move:
+    try:  # Upgrade cfg['scheduling']['dependencies']['graph']
+        if 'dependencies' in cfg['scheduling']:
+            msgOld = '[scheduling][dependencies][X]graph'
+            msgNew = '[scheduling][graph]X'
```

Comment:
I know this seems kind of trivial but camelCase is considered a style violation in Python https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

So, could you change this to `msg_old` etc.?

Reply:
Sorry, JavaScript habit!